docs/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine
+
+RUN apk update && apk add bash
+RUN apk update && apk add curl
+RUN apk update && apk add jq
```

Comment:
```suggestion
RUN apk add --no-cache bash curl jq
```

Reply:
Apologies - I hadn't meant to check that in.